src/Validation/EsValidation.php

```diff
@@ -51,11 +58,79 @@ public static function phone($check)
      * Checks a country specific identification number.
      *
      * @param string $check The value to check.
-     * @throws NotImplementedException Exception
      * @return bool Success.
      */
     public static function personId($check)
     {
-        throw new NotImplementedException(__d('localized', '%s Not implemented yet.'));
+        $checks = ['dni', 'nie', 'nif'];
+
+        foreach ($checks as $method) {
+            if (static::$method($check)) {
+                return true;
+            }
+        }
+
+        return false;
+    }
+
+    /**
+     * Only checks the DNI type personId.
+     *
+     * @param  string $check The value to check.
```

Comment:
Oh, ok. That's due to the docblocker plugin that I use.

They're removed now :smiley: